mkdocs.yml

```diff
@@ -0,0 +1,57 @@
+site_name: AllenNLP
+site_description: AllenNLP is a ..
+site_url: https://allennlp.org/
+
+theme:
+  name: material
+  palette:
+    primary: blue
+    accent: grey
+  logo: img/favicon.ico
+  favicon: img/favicon.ico
+  highlightjs: true
+  hljs_languages:
+  - python
+  - typescript
+  - json
+
+
+repo_name: allenai/allennlp
+repo_url: https://github.com/allenai/allennlp
+edit_uri: edit/master/docs/
+#google_analytics:
+#    - 'UA-133183413-1'
```

Comment:
Is this a safe id to have in source control?

Reply:
I think it's served publicly from the pages that register GA.  For example, it is on view-source:https://demo.allennlp.org/reading-comprehension (and we just copy and paste code from Google).